packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -243,20 +218,46 @@ function NavigationMenuSelector( {
 							>
 								{ __( 'Create new menu' ) }
 							</MenuItem>
-							{ isOffCanvasNavigationEditorEnabled && (
+							{ process.env.IS_GUTENBERG_PLUGIN ? ( // Show this if the plugin is active - previously used isOffCanvasNavigationEditorEnabled
```

Comment:
We only have this comment here but we use IS_GUTENBERG_PLUGIN in other places too. Isn't it self explanatory?

Reply:
The main reason is so that if we have to do a find/replace on this branch of the code in the future the places we need to change will be easier to find.